.github/workflows/image.yaml

```diff
@@ -40,3 +40,51 @@ jobs:
     secrets:
       registry_username: ${{ github.actor }}
       registry_password: ${{ secrets.CR_PAT }}
+
+  generate-init-kyverno-provenance:
+    needs: push-init-kyverno
+    permissions:
+      id-token: write # To sign the provenance.
+      packages: write # To upload assets to release.
+      actions: read #To read the workflow path.
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@9dc6318aedc3d24ede4e946966d30c752769a4f9
```

Comment:
Note: 
1. generator container is not released yet
2. builders need to be referenced by semver tag, see https://github.com/slsa-framework/slsa-github-generator#referencing-slsa-builders-and-generators

Reply:
Hi @laurentsimon - thanks for the note! 

I did some research on this and figured that:

- For 1, currently the workflow has `compile-generator` sets to `true`, so I guess it builds the generator from source.
- For 2, yes, the doc says "the generator MUST be referenced by a tag of the form @vX.Y.Z". However I failed to find how to define that in a workflow file. Here's a list of inputs but no "tag" option:

https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/container#workflow-inputs

Are you aware of any workaround to configure the tag?

Reply:
for 2: you can use `uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@v1.2.3` - verify the latest release and replace v1.2.3 accordingly. Please file an issue on the repo to ask for clarification on the documentation. Thanks!

Reply:
Thanks for the inputs @laurentsimon ! Will do.

Reply:
@laurentsimon when will the container provenance generator be officially released? I expected this might come as a KubeCon announcement but it must not.

Reply:
I think the current target is this month (Nov 2022)

Reply:
Thanks, Ian.

Shuting, can we resolve in the mean time or must we wait? Having proper attested provenance for Kyverno is an important initiative.

Reply:
We've had some stability issues recently and need to release a new version of the generator. We can update this to that version when we get it out.

Reply:
Can we simply watch for releases to be notified when we are able to fix our issues with SLSA provenance generation?

Reply:
I'll try to reach out after doing the release but checking for new releases is the official way.

Reply:
I set the watch for new releases 🤞

Reply:
There is a pre-release of 1.2.2 out now. Doesn't look like this claims GA for the generator though.

Reply:
Yeah, that's to fix the Rekor TUF issues. We'll definitely mention GA for generator in the release notes for a future release when it's ready.